src/Factory/Api/Mastodon/Account.php

```diff
@@ -94,13 +101,10 @@ public function createFromContactId(int $contactId, $uid = 0): \Friendica\Object
 	 */
 	public function createFromUserId(int $userId): \Friendica\Object\Api\Mastodon\Account
 	{
-		$publicContact = Contact::selectFirst([], ['uid' => $userId, 'self' => true]);
-
+		$contact       = DBA::selectFirst('account-user-view', [], ['uid' => $userId, 'self' => true]);
```

Comment:
I'd rather you rename this variable `$account` so that we don't confuse it with a `contact` row (like it used to be).
```suggestion
		$account       = DBA::selectFirst('account-user-view', [], ['uid' => $userId, 'self' => true]);
```